lib/keila_web/templates/user_admin/show_credits.html.heex

```diff
@@ -82,7 +82,7 @@
     </div>
     <br />
     <button class="button button--large button--cta">
-      <%= dgettext("admin", "Add Credits") %>
+      <%= dgettext("admin", "Add credits") %>
```

Comment:
I suppose we should also mirror this change in translation files.

https://github.com/marc-bouvier/keila/blob/31cfa3d1fc2e4052a2b9b9d0cf567284f9cbd6ed/priv/gettext/admin.pot#L27